components/contribution-flow/StepDetailsCrypto.js

```diff
@@ -55,18 +98,30 @@ const StepDetailsCrypto = ({ onChange, data }) => {
         {intl.formatMessage(messages['donationAmount'])}
       </Label>
       <StyledInputGroup
-        prepend={currencyType.labelWithoutImage}
+        prepend={cryptoCurrencyType.labelWithoutImage}
         type="number"
         inputMode="decimal"
         defaultValue={amount}
         onChange={({ target }) => {
           setAmount(target.value);
-          dispatchChange('amount', parseFloat(target.value));
+          if (target.value > 0) {
+            dispatchChange('amount', parseFloat(target.value));
+          }
         }}
         onBlur={() => setTouched(true)}
         autoFocus
         error={touched && amount <= 0 && intl.formatMessage(messages['invalidAmount'])}
       />
+      {convertedAmount !== null && convertedAmount > 0 && (
```

Comment:
Why not, but the following would have been enough 

```suggestion
      {convertedAmount && (
```

Reply:
This I intentionally did it this way as otherwise in the initial loading (and if user enter 0) we would have it a bit ugly with `0` appearing,

![Screenshot 2021-09-08 at 08-20-56 Contribute to Agora Collective - Open Collective](https://user-images.githubusercontent.com/12435965/132538071-a0a60780-23ed-4bcb-a47d-b2ec47096b7e.png)

Also when user enters negative amount it would appear like this,

![Screenshot 2021-09-08 at 08-20-43 Contribute to Agora Collective - Open Collective](https://user-images.githubusercontent.com/12435965/132538145-512c674d-67ed-4e9e-866b-e34173b8c676.png)

which seems weird; a negative monetary amount doesn't make sense. When we have the condition like, `convertedAmount !== null && convertedAmount > 0 &&` we explicitly handle these cases so that we don't show the approximate amount for `0` or negative amounts.

Reply:
We're not supposed to dispatch negative values anymore.

But that's fine.